po/wesnoth-wov/wesnoth-wov.pot

```diff
@@ -0,0 +1,2109 @@
+
+#. [leader]: id=Daepia, type=Elvish Enchantress
+#: data/campaigns/Wings_of_Victory/scenarios/08_Wesmere.cfg:78
+msgid "Daepia"
+msgstr ""
+
+#. [side]
+#: data/campaigns/Wings_of_Victory/scenarios/08_Wesmere.cfg:92
+#: data/campaigns/Wings_of_Victory/scenarios/12_Confrontation.cfg:112
+msgid "Saurians"
+msgstr ""
+
+#. [message]: speaker=Vank
+#: data/campaigns/Wings_of_Victory/scenarios/08_Wesmere.cfg:124
+msgid "There is game in those trees. This may be a good place to hunt."
+msgstr ""
+
+#. [message]: speaker=Galun
+#: data/campaigns/Wings_of_Victory/scenarios/08_Wesmere.cfg:128
+msgid "Yes. And the flight needs a night on the ground to rest its wings."
+msgstr ""
+
+#. [message]: speaker=Vank
+#: data/campaigns/Wings_of_Victory/scenarios/08_Wesmere.cfg:132
+msgid ""
+"Some of the game has two legs rather than four. They don’t quite look like "
+"our runners, though. They move better. And...Galun, I could swear some of "
+"the trees down there are <i>walking</i>!"
+msgstr ""
+
+#. [message]: speaker=Galun
+#: data/campaigns/Wings_of_Victory/scenarios/08_Wesmere.cfg:136
+msgid ""
+"I see them. Well, if they choose to fight us, our fire should burn them "
+"nicely. Call the hunt!"
+msgstr ""
+
+#. [message]: speaker=Galun
+#: data/campaigns/Wings_of_Victory/scenarios/08_Wesmere.cfg:144
+msgid "That was brisk."
+msgstr ""
+
+#. [message]: speaker=Vank
+#: data/campaigns/Wings_of_Victory/scenarios/08_Wesmere.cfg:148
+msgid ""
+"Yes, and there are more of the two-leggeds heading at us from every "
+"direction. I think I’m undercounting them — they’re cursed hard to spot in "
+"these woods."
+msgstr ""
+
+#. [message]: speaker=Galun
+#: data/campaigns/Wings_of_Victory/scenarios/08_Wesmere.cfg:152
+msgid ""
+"Set up a rotating overwatch. We’ll feed and rest here until the overwatch "
+"flyers spot more of them within a half-league, then fly north."
+msgstr ""
+
+#. [message]: speaker=Vank
+#: data/campaigns/Wings_of_Victory/scenarios/08_Wesmere.cfg:156
+#: data/campaigns/Wings_of_Victory/scenarios/11_Fire_Meets_Steel.cfg:227
+msgid "As you command!"
+msgstr ""
+
+#. [message]: speaker=Galun
+#. "scuttler" is a noun formed from the verb "to scuttle"
+#. It means to move quickly, and is often used of insects.
+#: data/campaigns/Wings_of_Victory/scenarios/08_Wesmere.cfg:208
+msgid "Wait, what is this? Scuttlers are approaching."
+msgstr ""
+
+#. [message]: speaker=Xirtrezyx
+#: data/campaigns/Wings_of_Victory/scenarios/08_Wesmere.cfg:219
+msgid ""
+"O great dragon, we have ssseen you strike at the hated humansss, and wish to "
+"follow you."
+msgstr ""
+
+#. [message]: speaker=Krenix
+#: data/campaigns/Wings_of_Victory/scenarios/08_Wesmere.cfg:223
+msgid "Thatsss not a dragon."
+msgstr ""
+
+#. [message]: speaker=Xirtrezyx
+#: data/campaigns/Wings_of_Victory/scenarios/08_Wesmere.cfg:227
+msgid ""
+"Sssilence! I can sssee that now that we are up close, but the dragon has "
+"beaten the humans and the elvesss, so do not disrespect the dragon."
```

Comment:
@sigurdfdragon, should this be "humansss"?

BTW, I love this line. It was great in the campaign, and it's still great looking at it multiple times.

Reply:
possibly